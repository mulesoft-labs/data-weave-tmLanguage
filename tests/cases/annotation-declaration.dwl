annotation AnnotationTarget(targets: Array<"Function" | "Parameter" | "Variable" | "Import">)