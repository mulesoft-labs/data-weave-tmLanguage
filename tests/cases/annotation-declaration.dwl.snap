>annotation AnnotationTarget(targets: Array<"Function" | "Parameter" | "Variable" | "Import">)
#^^^^^^^^^^ source.data-weave meta.directive.annot.dw storage.type.annotation.dw
#          ^ source.data-weave meta.directive.annot.dw
#           ^^^^^^^^^^^^^^^^ source.data-weave meta.directive.annot.dw entity.name.function.dw
#                           ^ source.data-weave meta.directive.annot.dw
#                            ^^^^^^^ source.data-weave meta.directive.annot.dw variable.parameter.dw
#                                   ^ source.data-weave meta.directive.annot.dw keyword.operator.declaration.dw
#                                    ^ source.data-weave meta.directive.annot.dw
#                                     ^^^^^ source.data-weave meta.directive.annot.dw support.type.dw
#                                          ^ source.data-weave meta.directive.annot.dw
#                                           ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw punctuation.definition.string.begin.dw
#                                            ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                             ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                              ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                               ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                 ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                  ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                   ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                    ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw punctuation.definition.string.end.dw
#                                                     ^ source.data-weave meta.directive.annot.dw
#                                                      ^ source.data-weave meta.directive.annot.dw keyword.operator.declaration.dw
#                                                       ^ source.data-weave meta.directive.annot.dw
#                                                        ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw punctuation.definition.string.begin.dw
#                                                         ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                          ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                           ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                            ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                             ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                              ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                               ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                 ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                  ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw punctuation.definition.string.end.dw
#                                                                   ^ source.data-weave meta.directive.annot.dw
#                                                                    ^ source.data-weave meta.directive.annot.dw keyword.operator.declaration.dw
#                                                                     ^ source.data-weave meta.directive.annot.dw
#                                                                      ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw punctuation.definition.string.begin.dw
#                                                                       ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                        ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                         ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                          ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                           ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                            ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                             ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                              ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                               ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw punctuation.definition.string.end.dw
#                                                                                ^ source.data-weave meta.directive.annot.dw
#                                                                                 ^ source.data-weave meta.directive.annot.dw keyword.operator.declaration.dw
#                                                                                  ^ source.data-weave meta.directive.annot.dw
#                                                                                   ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw punctuation.definition.string.begin.dw
#                                                                                    ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                                     ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                                      ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                                       ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                                        ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                                         ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw
#                                                                                          ^ source.data-weave meta.directive.annot.dw string.quoted.double.dw punctuation.definition.string.end.dw
#                                                                                           ^ source.data-weave meta.directive.annot.dw
#                                                                                            ^ source.data-weave meta.directive.annot.dw